src/main/java/frc/robot/EagleUtil.java

```diff
@@ -267,4 +349,34 @@ < getDistanceBetween(pose, getProcessorForAlliance())) {
     }
     return getProcessorForAlliance();
   }
+
+  public static double getOffsetArmAngle(CoralLevel level, Pose2d robotPose) {
+    double armAngle = level.armAngle;
+    int reefIndex = findClosestReefIndex(robotPose);
```

Comment:
we should move the reef index calculation below the Level 4 check, so we don't do calculations we don't need